common/script/api-v3/errors.js

```diff
@@ -0,0 +1,9 @@
+import CustomError from './customError';
+
+export class NotAuthorized extends CustomError {
```

Comment:
@crookedneighbor what if we used the errors with the httpCode in shared code too? I know it may seem a little weird but it becomes simpler to use them from the API, especially if they're more or less the same: NotAuthorized, NotFound and BadRequest as a catch all for the rest

Otherwise I can make a map of shared errors to http codes